base/version_git.sh

```diff
@@ -63,13 +63,13 @@ case $(uname) in
     date_string="$(/bin/date --date="@$git_time" -u '+%Y-%m-%d %H:%M %Z')"
     ;;
 esac
-if [ $(git describe --tags --exact-match 2> /dev/null) ]; then
```

Comment:
This can probably get rid of the `[ $() ]` altogether (replace with `> /dev/null`).